### DISCUSSION TITLE
Como usar un array de objetos como un prop

### DISCUSSION THREAD

**Discussion by Ismogar** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#533] [Q&A]
Estoy haciendo la practica de esta semana, tengo un componente para los sabores del pastel, lo repito 3 veces y le pongo un texto que engloba a los tres en plan "De que sabor quieres tu pastel?. Quiero convertir esto en un componente que pueda usar para los adornos, después de investigar un rato entiendo seria crear un nuevo componente que tenga como prop una lista de objetos y usar v-for para imprimir los elementos de la lista. 
Hice un intento usando lo que encontré [aquí](https://v2.vuejs.org/v2/guide/list.html#Mapping-an-Array-to-Elements-with-v-for) y [aquí](https://v2.vuejs.org/v2/guide/components-props.html?redirect=true#Passing-an-Array) pero no funciono, compilo el proyecto pero no me mostraba nada, en la consola me mostraba mensajes de error pero no los logre entender.
Puedo completar la practica sin hacer esto pero me gustaría experimentar un poco para saber que tanto se puede hacer con vue.

**Comment by Dfredude** [comment]
En la documentacion de mozilla viene justamente eso que quieres hacer: [Link](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Vue_rendering_lists)
Yo tambien intente hacerlo la primera vez y no me dejaba, fue super confuso, pero los ejemplos de esa pagina explican todo bien.
